test/dummy/app/controllers/application_controller.rb

```diff
@@ -1,5 +1,4 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
-  include ShopifyApp::LoginProtection
```

Comment:
I believe this needs be removed, because the [`UnauthenticatedHomeController` template](https://github.com/Shopify/shopify_app/blob/4fedb604e205927b54104f95d415e5fdbd2add59/lib/generators/shopify_app/home_controller/templates/unauthenticated_home_controller.rb) inherits from `ApplicationController` and so would have both `ShopifyApp::LoginProtection`  and `ShopifyApp::RequireKnownShop`.

cc @nelsonwittwer @teddyhwang